hivemq-edge/src/main/resources/config.xsd

```diff
@@ -59,6 +59,15 @@
                     </xs:sequence>
                 </xs:complexType>
             </xs:element>
+            <xs:element name="data-combiners" minOccurs="0">
```

Comment:
There should be a follo up ticket to change this into a proper XML definition later on.